### PR TITLE
task 9

### PR DESCRIPTION
**Pull request by Darina007**

### CODE REVIEW

**Comment by Darina007** [comment]
I have one question.
I want to make my project more convenient and readable and I really do not like the fact that there is only one css file, is it possible to logically split it into several files (for example, move everything related to the header to the header.css file ) or is it better not to do this?

**Comment by YauheniyaBasaranovich** [comment]
> I have one question.
> I want to make my project more convenient and readable and I really do not like the fact that there is only one css file, is it possible to logically split it into several files (for example, move everything related to the header to the header.css file ) or is it better not to do this?

That makes sense, go ahead 🙂

File: task3/UI/scripts/PostView.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        this.feedPosts.replaceChild(newPost, oldPost);
+    }
+
+    deletePost(id) {
+        let post = document.getElementById(id);
+        this.feedPosts.removeChild(post);
+    }
+
+    _fillPostData(newPost, postData) {
+        console.log(newPost);
```
Comment (YauheniyaBasaranovich) [inline task3/UI/scripts/PostView.js]: I think it might be removed

File: task3/UI/scripts/PostView.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        tag.textContent = "#" + tagText;
+        return tag;
+    }
+
+    _drawLike() {
+        let like = document.createElement('button');
+        like.className = "like";
+
+        let image = document.createElement('img');
+        image.src = "../img/like.png";
```
Comment (YauheniyaBasaranovich) [inline task3/UI/scripts/PostView.js]: There's no 'like.png' in 'img' folder 🙁

File: task3/UI/scripts/logic.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+function showFeed(filterConfig = {}, postModel, feedView) {
+    feedView.redrawPosts(postModel.getPage(0, 10, filterConfig));
+}
+
+function clear(postModel, feedView){
+    postModel.clear()
+    feedView.redrawPosts(postModel.getPage(0, 10));
+}
```
Comment (YauheniyaBasaranovich) [inline task3/UI/scripts/logic.js]: It might be good if you added functions to log in/out here as well.

**Review by YauheniyaBasaranovich — APPROVED** [review]
In general good job 👍  Two main things to fix: 
- add possibility to log in/out from console;
- for some reason select on the page doesn't open 😕 

Also just a proposal:
Structure of project is good now, but to make it more clear for any front-end developer I'd advice you to rename current 'tsk3' folder to 'UI', and current 'UI' - to 'src'. In this case all UI-related files will be grouped in 'UI', but configuration files (and node_modules) will be outside of source files.

File: task3/UI/scripts/PostView.js (hunk 1 above)
Comment (Darina007) [reply task3/UI/scripts/PostView.js]: Oops, forgot to delete 😅

Comment (Darina007) [reply task3/UI/scripts/PostView.js]: I fixed all the bugs, thanks again for your help.
Can I get the final grade for this task?)

Comment (YauheniyaBasaranovich) [reply task3/UI/scripts/PostView.js]: After I log in I can see 'add post' button, but log in button is still here:
Screen Shot 2021-05-05 at 3 54 52 PM

That's why grade is 9, but again you still can fix it.

Comment (Darina007) [reply task3/UI/scripts/PostView.js]: I apologize, my fault, now everything should work here (if not, then it definitely works in task 10 😄)
Thank you)

**Pull request merged**